### PR TITLE
[EDITORIAL] ChargeCategory: convert normative statement into bullet points

### PR DESCRIPTION
**Pull request by jpradocueva**
This PR is related to issue #606 and converts normative statements into bullet points.

### CODE REVIEW

**Review by ijurica — APPROVED** [review]

**Review by kk09v — APPROVED** [review]

File: specification/columns/chargecategory.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 Charge Category represents the highest-level classification of a charge based on the nature of how it is billed. Charge Category is commonly used to identify and distinguish between types of charges that may require different handling.
 
-The ChargeCategory column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) and MUST NOT be null. This column is of type String and MUST be one of the allowed values.
+The ChargeCategory column adheres to the following requirements:
+
+* The ChargeCategory column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) and MUST NOT be null.
+* This column is of type String and MUST be one of the allowed values.
```
Comment (flanakin) [inline specification/columns/chargecategory.md:8]: Shouldn't we split all requirements on unique rows?
```suggestion
* The ChargeCategory column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset).
* The ChargeCategory column MUST NOT be null.
* The ChargeCategory column MUST be of type String.
* The ChargeCategory column MUST be one of the allowed values.
```

Comment (flanakin) [reply specification/columns/chargecategory.md:8]: nit: Can we simplify this further and remove "This column"? This feels simpler and more direct to me.
```suggestion
* ChargeCategory MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset).
* ChargeCategory MUST NOT be null.
* ChargeCategory MUST be of type String.
* ChargeCategory MUST be one of the allowed values.
```

**Review by flanakin — APPROVED** [review]

File: specification/columns/chargecategory.md (hunk 1 above)
Comment (cnharris10) [reply specification/columns/chargecategory.md:8]: I think the initial editorial pass is to split sentences like Joaquin as done and then based on where the normative guidelines land, do 2nd pass

**Review by cnharris10 — APPROVED** [review]

**Comment by jpradocueva** [comment]
The PR was agreed on the Members' call on Dec 5.

**Pull request merged**